lib/alchemy/engine.rb

```diff
@@ -13,6 +13,7 @@
 require 'kaminari'
 require 'non-stupid-digest-assets'
 require 'request_store'
+require 'responders'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.